### PR TITLE
Log in/out

### PR DESCRIPTION
**Pull request by ExtremeMachine12**
Code is ready for review

### CODE REVIEW

File: src/recommenderapp/app.py [hunk 1]
```diff
@@ -17,29 +17,127 @@
...
 
 app = Flask(__name__)
 app.secret_key = "secret key"
 
 cors = CORS(app, resources={r"/*": {"origins": "*"}})
 
+# Connect to the database
+db.connect()
+# Initialize the database for users
+db.mutation_query('''
```
Comment (tanmaypardeshi) [inline src/recommenderapp/app.py:30]: Would it be better to make the code modular and move the queries in a different file? We might face issues with the linter if the code lines go beyond 1000

File: src/recommenderapp/app.py [hunk 2]
```diff
@@ -17,29 +17,127 @@
...
+    ret = {}
+    # Below is how we create a dictionary of the data sent from the input form
+    form_fields = dict((key, request.form.getlist(key)[0]) for key in list(request.form.keys()))
+    print(form_fields)
+
+    # query to see if user in the database already
+    user_query = f"SELECT * FROM users where email=?"
+    result = db.select_query(user_query, (form_fields["email"],))
+    print(result)
+    print('blehhhhh')
```
Comment (tanmaypardeshi) [inline src/recommenderapp/app.py:90]: Please remove the logging/debug statements if not needed.

File: src/recommenderapp/app.py [hunk 3]
```diff
@@ -17,29 +17,127 @@
...
+    ret = {}
+    # Below is how we create a dictionary of the data sent from the input form
+    form_fields = dict((key, request.form.getlist(key)[0]) for key in list(request.form.keys()))
+    print(form_fields)
+
+    # query to see if user in the database
+    user_query = f"SELECT * FROM users where email=? and password=?"
+    result = db.select_query(user_query, (form_fields["email"], form_fields["password"]))
+    print(result)
+    print('blehhhhh')
```
Comment (tanmaypardeshi) [inline src/recommenderapp/app.py:121]: Same as above

File: src/recommenderapp/templates/landing_page.html [hunk 4]
```diff
@@ -1,54 +1,27 @@
...
+    <p style="text-align: center; color: azure; font-size: 18px;"><br>
+    Discover personalized movie recommendations by selecting up to 5 of your favorite films.
+    <br>
+    Create a watchlist and have it conveniently sent to your email.
+    <br>
+    Enjoy movies at your own pace, on your terms.
+    </p><br>
+  <button id="getStartedButton" onclick="getStarted()" class="btn btn-primary mx-auto">Get Started!</button><br><br>
+    <div class="highlighted-section">
+      <p>Made with ❤️ by <a href="https://github.com/adipai/PopcornPicks" target="_blank"> PopcornPicks</a></p>
```
Comment (tanmaypardeshi) [inline src/recommenderapp/templates/landing_page.html:16]: I think this URL will be pointing to our repo now?

File: src/recommenderapp/templates/landing_page.html [hunk 5]
```diff
@@ -1,54 +1,27 @@
...
+      <a href="https://github.com/adipai/PopcornPicks/blob/master/LICENSE.md" target="_blank">MIT License Copyright (c) 2023 PopcornPicks</a>
```
Comment (tanmaypardeshi) [inline src/recommenderapp/templates/landing_page.html:17]: Same as above

**Review by tanmaypardeshi — CHANGES_REQUESTED** [review]
I have left some minor comments. Did not check the front-end code line by line as it looked good overall. I will approve the changes once the comments are resolved. Thanks!

File: src/recommenderapp/app.py (hunk 3 above)
Comment (ExtremeMachine12) [reply src/recommenderapp/app.py:121]: Fixed

File: src/recommenderapp/app.py (hunk 2 above)
Comment (ExtremeMachine12) [reply src/recommenderapp/app.py:90]: Fixed

File: src/recommenderapp/app.py (hunk 1 above)
Comment (ExtremeMachine12) [reply src/recommenderapp/app.py:30]: I think we can look into this later. Our app.py is at 200 lines of code right now, so I would be quite surprised if it went above 1000

File: src/recommenderapp/templates/landing_page.html (hunk 4 above)
Comment (ExtremeMachine12) [reply src/recommenderapp/templates/landing_page.html:16]: Fixed

File: src/recommenderapp/templates/landing_page.html (hunk 5 above)
Comment (ExtremeMachine12) [reply src/recommenderapp/templates/landing_page.html:17]: Fixed

File: src/recommenderapp/app.py (hunk 1 above)
Comment (tanmaypardeshi) [reply src/recommenderapp/app.py:30]: Okay!

**Review by tanmaypardeshi — APPROVED** [review]
Everything looks good to me!

**Pull request merged**